source/compiler/aslcompiler.l

```diff
@@ -461,6 +461,7 @@ NamePathTail                [.]{NameSeg}
 "PinGroup"                  { count (1); return (PARSEOP_PINGROUP); }
 "PinGroupConfig"            { count (1); return (PARSEOP_PINGROUPCONFIG); }
 "PinGroupFunction"          { count (1); return (PARSEOP_PINGROUPFUNCTION); }
+"ClockInput"                { count (1); return (PARSEOP_CLOCKINPUT); }
```

Comment:
I'm wondering if the (mis)location is deliberate (to split Pin*() resources instead of putting alphabetically).